.github/workflows/build_steps.yml

```diff
@@ -131,6 +148,9 @@ jobs:
           configurePreset: ${{env.ARCTIC_CMAKE_PRESET}}
           configurePresetAdditionalArgs: "['-DVCPKG_INSTALL_OPTIONS=--clean-after-build']"
           buildPreset: ${{env.ARCTIC_CMAKE_PRESET}}
+        env:
+          CMAKE_BUILD_PARALLEL_LEVEL: ${{ (matrix.os == 'linux' && inputs.job_type != 'build-python-wheels') && 1 || vars.CMAKE_BUILD_PARALLEL_LEVEL}}
+          CXX: /opt/rh/gcc-toolset-10/root/bin/g++
```

Comment:
This might mess up windows builds

Reply:
yeah good catch, it is not needed in general, so I've removed it.